### PR TITLE
Coswara dataset

### LINKED ISSUE #32: Coswara Dataset
**Claim Dataset:** [Coswara](https://github.com/iiscleap/Coswara-Data)

**About Dataset:** The COVID-19 pandemic presents global challenges transcending boundaries of country, race, religion, and economy. The current gold standard method for COVID-19 detection is reverse transcription-polymerase chain reaction (RT-PCR) testing. However, this method is expensive, time-consuming, and violates social distancing. Also, as the pandemic is expected to stay for a while, there is a need for an alternate diagnosis tool that overcomes these limitations and is deployable at a large scale. The prominent symptoms of COVID-19 include cough and breathing difficulties. We foresee that respiratory sounds, when analyzed using machine learning techniques, can provide useful insights, enabling the design of a diagnostic tool. 

Towards this, the paper presents an early effort in creating (and analyzing) a database, called Coswara, of respiratory sounds, namely, cough, breath, and voice. The sound samples are collected via worldwide crowdsourcing using a website application. The curated dataset is released as open access. As the pandemic is evolving, data collection and analysis are a work in progress. We believe that insight from the analysis of Coswara can be effective in enabling sound-based technology solutions for point-of-care diagnosis of respiratory infection, and in the near future, this can help to diagnose COVID-19.

### PR DESCRIPTION
**Pull request by CyberFlameGO**
@mertbozkir contribute the README to the `feat/coswara` branch of my fork, or DM me the README you want in here. Here's the (empty at the moment) DAGsHub repo: `https://dagshub.com/CyberFlame/Coswara-Data`

Closes #32

### CODE REVIEW

**Comment by CyberFlameGO** [comment]
image
image
image
image

**Comment by CyberFlameGO** [comment]
This PR is ready to be merged

**Comment by nirbarazida** [comment]
Hi @CyberFlameGO 

Please add a link to the DagsHub repo in the README file.

File: Coswara/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+## <strong>Can I know the individuals maintaining this project?</strong>
+Yes, we are a team of Professors, PostDocs, Engineers, and Research Scholars affiliated with the Indian Institute of Science, Bangalore (India). [Sriram Ganapathy](http://leap.ee.iisc.ac.in/sriram/), Assistant Professor, Dept. Electrical Engineering, IISc is the Principal Investigator of this project.
+
+Current Members: Debarpan Bhattachrya, Debottam Dutta, Neeraj Kumar Sharma, Prasanta Kumar Ghosh, Srikanth Raj Chetupalli, Sriram Ganapathy
+
+Past Members: Anand Mohan, Ananya Muguli, Prashant Krishnan, Rohit Kumar, Shreyas Ramoji
+
+---
+
```
Comment (CyberFlameGO) [inline Coswara/README.md:103]:
```suggestion

The DAGshub repository is [here](https://dagshub.com/CyberFlameGO/Coswara)
```

**Comment by CyberFlameGO** [comment]
My bad haha, I thought it was done earlier

**Comment by nirbarazida** [comment]
@CyberFlameGO The link is broken - I get 404

File: Coswara/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+The DAGshub repository is [here](https://dagshub.com/CyberFlameGO/Coswara)
```
Comment (CyberFlameGO) [inline Coswara/README.md:104]:
```suggestion
The DAGshub repository is [here](https://dagshub.com/CyberFlameGO/Coswara-Data)
```

**Comment by CyberFlameGO** [comment]
My apologies, must've messed it up. Fixed

File: Coswara/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+## <strong>Can I know the individuals maintaining this project?</strong>
+Yes, we are a team of Professors, PostDocs, Engineers, and Research Scholars affiliated with the Indian Institute of Science, Bangalore (India). [Sriram Ganapathy](http://leap.ee.iisc.ac.in/sriram/), Assistant Professor, Dept. Electrical Engineering, IISc is the Principal Investigator of this project.
+
+Current Members: Debarpan Bhattachrya, Debottam Dutta, Neeraj Kumar Sharma, Prasanta Kumar Ghosh, Srikanth Raj Chetupalli, Sriram Ganapathy
+
+Past Members: Anand Mohan, Ananya Muguli, Prashant Krishnan, Rohit Kumar, Shreyas Ramoji
+
+---
+
+The DAGshub repository is [here](https://dagshub.com/CyberFlameGO/Coswara-Data)
```
Comment (CyberFlameGO) [inline Coswara/README.md:104]:
```suggestion
The DAGshub repository is [here](https://dagshub.com/CyberFlame/Coswara-Data)
```

**Review by CyberFlameGO — COMMENTED** [review]
Shh

**Pull request merged**